bin/link-changelog

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# link creates a link to changelog for the package's version on https://pub.dev.
+#
+# It assumes that the git tag starts with "v".
+
+package_name="${1:-}"
+tag="${2:-}"
+prefix="${3:-}"
+
+if [ -z "$package_name" ]; then
+    echo "error: missing package name"
+    exit 1
+
+fi
+
+if [ -z "$tag" ]; then
+    echo "error: missing tag"
+    exit 1
+fi
+
+if [ -n "$prefix" ]; then
+    tag="${tag//$prefix/}"
```

Comment:
This will remove all instances of `$prefix` from the tag, not only at the beginning as the name would suggest. How about something like this instead?

```suggestion
    tag="${tag#$prefix}"
```

Reply:
❤️